platforms/beepy/src/i2c_puppet.rs

```diff
@@ -260,11 +310,21 @@ impl I2cPuppetServer {
             .map_err(RegistrationError::NoI2cPuppet)?;
         tracing::info!("i2c_puppet firmware version: v{major}.{minor}");
 
+        let cfg = reg::Cfg::new()
+            .with(reg::Cfg::KEY_INT, true)
+            .with(reg::Cfg::USE_MODS, true)
+            .with(reg::Cfg::OVERFLOW_INT, true);
```

Comment:
```suggestion
            .with(reg::Cfg::OVERFLOW_INT, true)
            // overwrite older keypresses when the FIFO is full. 
            // since we only poll the keyboard when there are active 
            // subscriptions, enable this setting so that the
            // FIFO doesn't fill up with ancient keypresses. 
            .with(reg::Cfg::OVERFLOW_ON, true);
```